src/components/FormValidator.js

```diff
@@ -73,4 +73,10 @@ export default class FormValidator {
     });
```

Comment:
Please, delete `this._disableSubmitButton();` from line 72

You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.